.github/workflows/full-android.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
         if: github.actor != 'restyled-io[bot]'
 
         container:
-            image: ghcr.io/project-chip/chip-build-android:108
```

Comment:
Needs to be 113 now, as we got some merge intersection

Reply:
Also would be nice to update all references from 112 to 113 to get everyone consistent.

See what #37356 did